<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registration</class>
 <widget class="QMainWindow" name="Registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registration</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="start" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1281</width>
      <height>691</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>179</y>
       <width>1261</width>
       <height>261</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="registration_label2">
        <property name="text">
         <string>Registration</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="who_are_you">
        <property name="text">
         <string>Who are you?</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="worker_btn">
          <property name="text">
           <string>Worker</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="guest_btn">
          <property name="text">
           <string>Guest</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;-Go Back</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="worker" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1281</width>
      <height>691</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>180</y>
       <width>1261</width>
       <height>381</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="regoster_label">
        <property name="text">
         <string>Registration of worker</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="name_surname">
        <property name="placeholderText">
         <string>name and surname</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="position">
        <property name="placeholderText">
         <string>position</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="contacts">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>contacts</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nickname">
        <property name="placeholderText">
         <string>nickname</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="password">
        <property name="placeholderText">
         <string>password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="password_rep">
        <property name="placeholderText">
         <string>repeat password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="create">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Create Account</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="back">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;-Go Back</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="guest" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1281</width>
      <height>691</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>180</y>
       <width>1261</width>
       <height>381</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="regoster_label_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Registration of guest</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="name_surname_guest">
        <property name="placeholderText">
         <string>name and surname</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="contacts_guest">
        <property name="placeholderText">
         <string>contacts</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="description_guest">
        <property name="placeholderText">
         <string>description</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nickname_guest">
        <property name="placeholderText">
         <string>nickname</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="password_guest">
        <property name="placeholderText">
         <string>password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="password_rep_guest">
        <property name="placeholderText">
         <string>repeat password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="create_2">
        <property name="text">
         <string>Create Account</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;-Go Back</string>
     </property>
    </widget>
   </widget>
   <zorder>guest</zorder>
   <zorder>worker</zorder>
   <zorder>start</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
